Formula/libpsl.rb

```diff
@@ -0,0 +1,41 @@
+class Libpsl < Formula
+  desc "C library for the Publix Suffix List"
+  homepage "https://github.com/rockdaboot/libpsl"
+  url "https://github.com/rockdaboot/libpsl/releases/download/libpsl-0.14.0/libpsl-0.14.0.tar.gz"
+  sha256 "e8c794bcedf45c87dee4810570bc62ba7130217887e34e164e3eadc2259f2efb"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "md5sha1sum" => :build
+  depends_on :python2 => :build
+  depends_on "icu4c"
+  # A build script (psl-make-dafsa) requires Python 2.7.
+  depends_on :python => :build if MacOS.version <= :snow_leopard
+  depends_on "icu4c"
+
+  # OS X doesn't follow PEP-394, so change psl-make-dafsa's shebang line to use Python 2.
+  patch :DATA
+
+  def install
+    system "autoconf"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    test_output = shell_output("#{bin}/psl --print-unreg-domain www.example.com").strip
+    assert_equal "www.example.com: com", test_output
+  end
+end
+
+__END__
+--- a/src/psl-make-dafsa
++++ b/src/psl-make-dafsa
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python2
++#!/usr/bin/env python
```

Comment:
I thought we found that it wasn't possible to make the shebang cross-platform?  We're doing this already with 
- [`emscripten.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/emscripten.rb)
- [`git-remote-hg.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/git-remote-hg.rb)
- [`gst-validate.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gst-validate.rb)
- [`src.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/src.rb)
- [`yle-dl.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/yle-dl.rb)